lib/Event/RowAddedEvent.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OCA\Tables\Event;
+
+use OCA\Tables\Db\Row2;
+use OCP\EventDispatcher\Event;
+
+final class RowAddedEvent extends Event {
+	public function __construct(protected Row2 $row) {
+		parent::__construct();
+	}
+
+	public function getRow(): Row2 {
+		return $this->row;
```

Comment:
@blizzz What's your thought on just returning the entity directly here?

I'm unsure if we shall expose the internal row entity through events, rather expose specific getters or use a serializable event as https://github.com/nextcloud/tables/issues/1103 mentioned

Reply:
I also have reservations about exposing the Entity. Turning this into a minified (and so serializable) model instance might could be a way. The Entity is can actually be json-serialized, that could be a base.

Then, in principle, whatever we are doing, we are de-facto exposing a Model to the outside, so setting standards as in an API, and have to keep them stable. That's generally fine, but would be a firstee at least in this app.

Reply:
Just read the note about `OCP\EventDispatcher\IWebhookCompatibleEvent` which is already in master meanwhile. But this is yet another `jsonSerialize()` style method. In a way i still tend to a public model class that itself is just json serializable and we return the output here. On the other hand, this can be already done with the DB Entity.

I also noticed that we are actually exposing the DB Entities already in other events that are also released already. Including Row2.